openapi2beans/pkg/generator/JavaStructures.go

```diff
@@ -50,24 +52,31 @@ func NewJavaClass(name string, description []string, javaPackage *JavaPackage, d
 // order is:
 // boolean > int > double > String > other
 func (class JavaClass) Sort() {
-	sort.SliceStable(class.DataMembers, func(i int, j int) bool { return isDataMemberLessThanComparison(class.DataMembers[i], class.DataMembers[j]) })
-	sort.SliceStable(class.ConstantDataMembers, func(i int, j int) bool { return isDataMemberLessThanComparison(class.ConstantDataMembers[i], class.ConstantDataMembers[j]) })
+	sort.SliceStable(class.DataMembers, func(i int, j int) bool {
+		return isDataMemberLessThanComparison(class.DataMembers[i], class.DataMembers[j])
+	})
+	sort.SliceStable(class.ConstantDataMembers, func(i int, j int) bool {
+		return isDataMemberLessThanComparison(class.ConstantDataMembers[i], class.ConstantDataMembers[j])
+	})
 	if class.RequiredMembers != nil {
 		class.RequiredMembers[0].IsFirst = false
 	}
-	sort.SliceStable(class.RequiredMembers, func(i int, j int) bool { return isDataMemberLessThanComparison(class.RequiredMembers[i].DataMember, class.RequiredMembers[j].DataMember) })
+	sort.SliceStable(class.RequiredMembers, func(i int, j int) bool {
+		return isDataMemberLessThanComparison(class.RequiredMembers[i].DataMember, class.RequiredMembers[j].DataMember)
+	})
 	if class.RequiredMembers != nil {
 		class.RequiredMembers[0].IsFirst = true
 	}
```

Comment:
because we're sorting 3 seperate lists, and because member and comparisonMember feels very clunky for a sort function, please let me know if you think it should be more verbose though.